src/tests/ftest/server/cpu_usage.yaml

```diff
@@ -1,7 +1,32 @@
 hosts:
   test_servers:
     - server-A
-timeout: 80
+  test_clients:
+    - client-A
+timeout: 130
 server_config:
   servers:
+    # 16 targets and 16 nr_xs_helpers are the requirements.
     targets: 16
+    nr_xs_helpers: 16
+    bdev_class: nvme
+    bdev_list: ["0000:00:00.0"]
+    scm_class: dcpm
+    scm_list: ["/dev/pmem0"]
+    scm_mount: /mnt/daos0
+    interface:
+      fabric_iface: ib0
```

Comment:
Was there a reason to specify this rather than just let the interface auto-detection code figure out to use it if it's available?

Reply:
We wanted to test the CPU usage with ib0. I think launch.py or other setup code will auto-detect and use faster network interface, but I wanted to be explicit since it's kind of a requirement.

Reply:
But it will be automatically/implicitly used on a platform where it exists, right?  That achieves your goal doesn't it?  So if you want `ib0` to be used, you'd ensure the test runs on a platform with an `ib0` on it couldn't you?